analyzers/src/SonarAnalyzer.Utilities/Rules.Description/S4784_cs.html

```diff
@@ -85,6 +85,4 @@ <h2>See</h2>
   </li>
   <li> OWASP Regular expression Denial of Service - ReDoS </li>
 </ul>
-<h2>Deprecated</h2>
-<p>This rule is deprecated; use {rule:csharpsquid:S2631} instead.</p>
```

Comment:
I think we should keep this message. It's relevant for the user. Another tool issue?

Reply:
This rule was marked as `ready` during migration from Jira (see this [thread](https://sonarsource.slack.com/archives/CFSLVKNBB/p1624024528158900) ) while it should remain deprecated. So the tool itself is probably fine.

I just missed the fact that this should be undone.